CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
 # Changelog
 
-## [0.0.3 UNRELEASED]
+## [0.1.0]
 
 ### Added
 
 - Healthcheck test
```

Comment:
Does somebody know if this belongs to a issue?

Reply:
no, its ok.

Reply:
I found it. It's #4